src/components/Pages/Login.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <PageMenubar />
+  <div class="login">
+    <img class="logo" alt="app logo" src="/assets/logo.png" />
+    <h2>Login to Proceed...</h2>
+    <br />
+    <div class="input-field">
+      <div class="email-input">
+        <span class="p-float-label">
+          <InputText
+            id="email1"
+            v-model="email"
+            type="text"
+            class="w-full mb-3"
+          />
+          <label for="inputtext">Enter Email</label>
+        </span>
+      </div>
+      <span class="p-float-label">
+        <InputText
+          id="email1"
+          v-model="password"
+          type="text"
+          class="w-full mb-3"
+        />
+        <label for="inputtext">Enter Password</label>
+      </span>
+    </div>
+    <br />
+
+    <Button class="p-button" label="Submit" v-on:click="login"
+      ><span class="submit-button">Login</span></Button
+    >
+    <p>
+      <router-link to="/sign-up">Sign Up</router-link>
+    </p>
+  </div>
+</template>
+
+<script>
+import InputText from "primevue/inputtext"
+import Password from "primevue/password"
+import Button from "primevue/button"
+import "primevue/resources/themes/lara-light-indigo/theme.css"
```

Comment:
We are not using this theme. Please update it.

Reply:
Also, add a comment somewhere that these imports will be removed in the future.

Reply:
while removing this theme file, facing same issue of styling. Comment added for these files.